.github/workflows/go.yml

```diff
@@ -111,10 +111,12 @@ jobs:
       - name: Build mackerel-agent-plugins
         run: |
           cd wix
-          for p in $(./pluginlist.sh)
+          plugins=( $(./pluginlist.sh) ) || exit 1
```

Comment:
We may need `IFS=` here.

But, for now, this is ok because **pluginlist.sh** does not contain any spaces in each plugins name.